Jenkinsfile

```diff
@@ -19,7 +19,6 @@ nodeJob {
   runBookeeping = true
 
   //static asset upload
-  runPrivateNpmPublish = false
```

Comment:
true by default.

Reply:
i'm always in favor of explicit value (vs default values, which requires finding where the default value is defined to infer the final behavior).

Reply:
worth discussing, as this would introduce a lot of noise.